locations/spiders/maggianos_little_italy.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+import json
+import re
+
+import scrapy
+
+from locations.hours import OpeningHours
+from locations.items import Feature
+
+
+class MaggianosLittleItalySpider(scrapy.Spider):
+    name = "maggianos_little_italy"
+    item_attributes = {"brand": "Maggiano's Little Italy", "brand_wikidata": "Q6730149"}
+    allowed_domains = ["maggianos.com"]
+    start_urls = [
+        'https://www.maggianos.com/api//restaurant-data/',
+    ]
+
+    def parse_hours(self, hours):
+        opening_hours = OpeningHours()
+
+        for hour in hours:
+            day = hour["day_name"][:2]
+            opening_hours.add_range(day, open_time=hour["open_time"], close_time=hour["end_time"])
+
+        return opening_hours.as_opening_hours()
```

Comment:
Return just `opening_hours` instead of the textual representation generated through `opening_hours.as_opening_hours()`.